tutorials-v5/time-evolution/008_brmesolve_time_dependence.md

```diff
@@ -86,8 +86,9 @@ $$ J(\omega, t) = \kappa * e^{-t} \quad \text{for} \; \omega \geq 0$$
 ```python
 # setup dissipation
 kappa = 0.2
-a_ops = [[a + a.dag(), "{kappa}*exp(-t)*(w>=0)".format(kappa=kappa)]]
-
+a_ops = [
+    ([a+a.dag(), f'sqrt({kappa}*exp(-t))'], '(w>=0)')
+]
```

Comment:
I just ran the notebook, and for me the test passes both with and without the `sqrt`. Hum.

Reply:
after some poking around, i guess the logic of the docs is that the time-dependence is absorbed into the ''operator'' part, and eventually treated as a QobjEvo, hence the square root (since it would be squared if pulled out again and put in the power spectrum).  

Am not sure why the alternative now doesn't work, with just absorbing all the time-dependence in to the power spectrum.   Only change I can see is this PR from @Ericgig  https://github.com/qutip/qutip/commit/c5566a416c728596d1d8c273424e7ab7bf2187cc

@Ericgig should something `a_ops = [[a + a.dag(), "{kappa}*exp(-t)*(w>=0)".format(kappa=kappa)]]` work still?   should it ever have worked?  Time-dependence a_ops is not something i ever used in practise, so am not sure.

Reply:
just for the record, `a_ops = [[QobjEvo(a + a.dag()), "{kappa}*exp(-t)*(w>=0)".format(kappa=kappa)]]` seems to work (and is suggested by the doc strings... i think).  Anyways, I think this makes sense now (?), but maybe the documentation can explain this option a bit better.